settings.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Default options
+VALID_ROS_DISTROS="kinetic melodic noetic foxy dashing"
+ROS_DISTRO="noetic"
+VOLUMES_FOLDER=$HOME/dev_ws/volumes/
```

Comment:
VALID_ROS_DISTROS should be a constant, not a config variable. I think ROS_DISTRO fits better as a positional argument. VOLUMES_FOLDER could also be an argument for the script, but looking at this file now I see it would be nice to have a default value fixed as the user wants...

So, now I think it is not so bad to have a settings file, especially for the VOLUMES_FOLDER option. If you like the idea of keeping it, I suggest renaming the file to something like `['build-image' | ''].[config | settings]` to indicate it is not a script.